keps/prod-readiness/sig-node/3741.yaml

```diff
@@ -0,0 +1,7 @@
+kep-number: 3741
```

Comment:
Instead of opening a new issue - we should reopen https://github.com/kubernetes/enhancements/issues/693 and reuse it (as it has more history inside).

Reply:
I referred to the recommendation here: https://github.com/kubernetes/enhancements/issues/693#issuecomment-815813702 and opened a new issue. Let me know if you would prefer the issue to be reopened.

Reply:
Ah I see, it was recommended that the details be captured as a comment [here](https://github.com/kubernetes/enhancements/issues/693#issuecomment-816739631). Is that something you think would be more preferrable?

Reply:
Yeah - we should reuse the same one. I reopened it - you should be able to use it now.
[Also if you need anything, feel free to ping me.]

Reply:
Thanks!